### ISSUE TITLE
Running tests does not read from resources directory

### ISSUE THREAD

**Issue by leszekgruchala** [scalacenter/bloop#102]
Bloop version: `66018e0a`

I have Akka-http project and `application.conf` in `src/main/resources` as well some overrided properties in `src/test/resources` directories. When I run tests in sbt all is fine, but with bloop I get errors about missing configuration:

```
com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'akka'
  at com.typesafe.config.impl.SimpleConfig.findKeyOrNull(SimpleConfig.java:156)
  at com.typesafe.config.impl.SimpleConfig.findKey(SimpleConfig.java:149)
  at com.typesafe.config.impl.SimpleConfig.findOrNull(SimpleConfig.java:176)
  at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:188)
  at com.typesafe.config.impl.SimpleConfig.find(SimpleConfig.java:193)
  at com.typesafe.config.impl.SimpleConfig.getString(SimpleConfig.java:250)
  at akka.actor.ActorSystem$Settings.<init>(ActorSystem.scala:314)
  at akka.actor.ActorSystemImpl.<init>(ActorSystem.scala:650)
  at akka.actor.ActorSystem$.apply(ActorSystem.scala:244)
  at akka.actor.ActorSystem$.apply(ActorSystem.scala:287)
```

**Comment by jvican** [comment]
This will be in for our next release, that will happen at the end of the week. Thanks for reporting @leszekgruchala :heart:.

**Comment by Duhemm** [comment]
This was fixed in #149
